src/index.ts

```diff
@@ -55,5 +57,44 @@ export default function createReScriptPlugin(): Plugin {
         await launchReScript(command === 'serve' || Boolean(build.watch));
       }
     },
+    config: () => ({
+      server: {
+        watch: {
+          // Only watch for changes to the actual compiled js files, except the
+          // ".compiler.log" file which is needed for showing the error overlay.
+          // JS is not outputted when the compiler errors, but the rescript files
+          // would still trigger an update. That is why they are ignored here.
+          ignored: [
+            '**/*.res',
+            '**/*.resi',
+            '**/lib/bs',
+            '!**/lib/bs/.compiler.log',
+          ],
+        },
+      },
+    }),
+    configureServer(server) {
+      // Manually find and parse log file after server start since
+      // initial compilation does not trigger handleHotUpdate.
+      glob('**/lib/bs/.compiler.log', {}, (globError, files) => {
```

Comment:
It seems like we'd only have one `.compiler.log`, is it necessary to use `glob` to find it?

Reply:
No, you are right, there is only one `.compiler.log`. I was only using it for the wildcard search, since I was unsure if the `lib` folder is always in the same location. But now that I think about it, I guess it would always be in `cwd` of the executing script. What do you think?

Reply:
5729afa